sentry/src/main/java/io/sentry/protocol/SentryStackTrace.java

```diff
@@ -40,21 +41,27 @@
    * Required. A non-empty list of stack frames. The list is ordered from caller to callee, or
    * oldest to youngest. The last frame is the one creating the exception.
    */
-  private List<SentryStackFrame> frames;
+  private @Nullable List<SentryStackFrame> frames;
   /**
    * Register values of the thread (top frame).
    *
    * <p>A map of register names and their values. The values should contain the actual register
    * values of the thread, thus mapping to the last frame in the list.
    */
-  private Map<String, String> registers;
+  private @Nullable Map<String, String> registers;
+
+  /**
+   * This flag indicates that this stack trace is captured at an arbitrary moment in time and this
+   * would affect the quality of grouping, Sentry will special case if this is set to true.
```

Comment:
`m`: Can we maybe explain more in detail how this would affect grouping and how Sentry is going to special case this?

Reply:
it's not defined yet, backend implementation is not even started.